src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -145,15 +140,15 @@ def update_license_file(arg_dict: dict) -> int:
     """
     # Get location of LICENSE file in the repository the hook runs on
     git_root = arg_dict["git_repo"].git.rev_parse("--show-toplevel")
-    repo_license_loc = os.path.join(git_root, "LICENSE").replace(os.sep, "/")
-    save_repo_license = shutil.copyfile(repo_license_loc, f"{repo_license_loc}_save")
+    repo_license_loc = Path(git_root) / "LICENSE"
+    save_repo_license = Path(shutil.copyfile(repo_license_loc, f"{repo_license_loc}_save"))
 
     # Get the location of MIT.txt in the hook's assets folder
-    hook_loc = pathlib.Path(__file__).parent.resolve()
-    hook_license_file = os.path.join(hook_loc, "assets", "LICENSES", f"{DEFAULT_LICENSE}.txt")
+    hook_loc = Path(__file__).parent.resolve()
+    hook_license_file = Path(hook_loc) / "assets" / "LICENSES" / f"{DEFAULT_LICENSE}.txt"
```

Comment:
```suggestion
    hook_license_file = hook_loc / "assets" / "LICENSES" / f"{DEFAULT_LICENSE}.txt"
```
No need - ``hook_loc`` after being resolved is still a Path object